Makefile

```diff
@@ -51,7 +51,7 @@ vet:
 	go vet ./...
 
 .PHONY: test
-test: lint vet
```

Comment:
why move this out of `test`?

Will this impact workflows other than the main `test` one?

Reply:
i rolled this back